src/components/DpInput/DpInput.vue

```diff
@@ -30,124 +29,172 @@
       :required="required"
       :autocomplete="autocomplete !== '' ? autocomplete : null"
       :size="(size && size > 0) ? size : null"
-      v-model="currentValue"
-      @blur="$emit('blur', currentValue)"
+      v-model="props.value"
+      @blur="$emit('blur', props.value)"
       @focus="$emit('focus')"
-      @input="$emit('input', currentValue)"
+      @input="$emit('input', props.value)"
```

Comment:
If we want to use `v-model` in the parent component, i.e. when using `DpInput`, we need to emit an `'update:modelValue'` event instead of an `'input'` event.

Reply:
right. Indeed. would it be legit in your opinion to fire both events? Otherwise, I have to ensure, that  @input is not used for DpInput, which I doubt and would be a bit hard to figure out.

Reply:
eebb013e6be654cf75414d68b425f60c7793f273

Reply:
nice!